happybase/table.py

```diff
@@ -222,7 +222,7 @@ def cells(self, row, column, versions=None, timestamp=None,
     def scan(self, row_start=None, row_stop=None, row_prefix=None,
              columns=None, filter=None, timestamp=None,
              include_timestamp=False, batch_size=1000, scan_batching=None,
-             limit=None, sorted_columns=False):
+             limit=None, sorted_columns=False, reversed=False):
```

Comment:
any reason you used `reversed`, not `reverse`? python itself uses `reverse` for this purpose, e.g. `sorted(, ...reverse=...)` and `list.sort(reverse=True)`. `reversed()` is the name of the built-in function.